Makefile

```diff
@@ -38,7 +39,8 @@ SHFMT = $(TOOLS_BIN_DIR)/shfmt
 LINTER = $(TOOLS_BIN_DIR)/golangci-lint
 release: clean test build package-rpm package-deb package-win package-darwin
 
-nightly-release: release
+nightly-release: clean test build package-rpm package-deb package-win
```

Comment:
We can't build mac on linux since the cpu plugin will not work. https://github.com/aws/telegraf/blob/aws-cwa-release-1-23-0/patches/gopsutil/v3/cpu/cpu_darwin_nocgo.go

Reply:
I know. We only leverage `clean test build` from `nightly-release-mac `. We are not using the mac build

Reply:
That would still include the package-darwin if we make this change.

Reply:
Added prepackage step.